lib/edgestats.js

```diff
@@ -27,6 +27,9 @@ function waitForEdgeIceCompletion(peerConnection) {
   if (peerConnection.iceConnectionState === 'new'
     || peerConnection.iceConnectionState === 'checking'
     || peerConnection.iceConnectionState === 'connected') {
+    if (typeof iceCompletionDeferreds === 'undefined') {
```

Comment:
@syerrapragada Nitpick: `iceCompletedDeferreds = iceCompletedDeferreds || new WeakMap();`?
Also, add a note here as to why we are lazy-instantiating?